### PR TITLE
2 header

### LINKED ISSUE #2: [Header] Implement and style header
- [ ] Desktop version
Screenshot 2023-07-21 at 1 06 37 PM

- [ ] Mobile version
Screenshot 2023-07-21 at 1 07 07 PM

- [ ] In Mobile view, create multi level navigation menu and move the hamburger to the right side.

### PR DESCRIPTION
**Pull request by ParthasharathiB**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #2 

Test URLs:
- Before: https://main--fonterra-fernleaf--hlxsites.hlx.page/en
- After: https://2-header--fonterra-fernleaf--hlxsites.hlx.page/en

### CODE REVIEW

**Comment by mkbansal1** [comment]
@ParthasharathiB noticed several issues:

1. Very big logo image is used, which shows in actual size for a fraction of a second. Use the correct logo image.
2. Logo image is not clickable
3. Font and styling is not matching live site
image
image

4. search icon is broken
image

5. Mobile navigation style is also different from live site
6. mobile nav - Arrow icon is broken
7. Lint check failed

**Comment by ParthasharathiB** [comment]
Hi @mkbansal1 ignore this PR,  this one is not finalised one. Still i am working on this. I pushed and created PR due to conflict issue. I will let you know once done my changes.

**Comment by asthabh23** [comment]
@ParthasharathiB, added Draft PR label on this PR, once it is ready to review you can remove the label

**Review by mkbansal1 — APPROVED** [review]

File: blocks/header/header.css [hunk 1]
```diff
@@ -1,89 +1,207 @@
...
 }
 
-header nav[aria-expanded="true"] {
-  grid-template:
-    'hamburger brand' var(--nav-height)
-    'sections sections' 1fr
-    'tools tools' var(--nav-height) / auto 1fr;
-  overflow-y: auto;
-  min-height: 100vh;
+span.icon.icon-search svg {
```
Comment (noormd08) [inline blocks/header/header.css:34]: Please keep the parent reference if this is specific to header (or) move this icon to a generic file

File: blocks/header/header.css [hunk 2]
```diff
@@ -1,89 +1,207 @@
...
 @media (min-width: 900px) {
   header nav {
     display: flex;
     justify-content: space-between;
+    padding-top: 3px;
+    padding-right: 28px;
+    font-family: var(--global-quaternary-font-family);
+  }
+
+  span.icon.icon-search {
```
Comment (noormd08) [inline blocks/header/header.css:107]: Same as above.

File: blocks/header/header.css [hunk 3]
```diff
@@ -198,77 +408,110 @@ header nav .nav-sections ul > li > ul > li {
...
-header nav .nav-tools {
-  grid-area: tools;
+
+/* search icon */
+.icon-search {
+  height: 30px;
+  width: 30px;
 }
+
+@media only screen and (max-width: 900px) {
```
Comment (noormd08) [inline blocks/header/header.css:513]: Multiple media queries with same dimension are presented. Can this be combined?

**Pull request merged**